_includes/tags.html

```diff
@@ -19,19 +19,19 @@
                        using the literal string '/blog/'. Path is used to
                        create the filtered URL: {{ path }}?filter_tags={{ tag }}.
 
-   display_header:     The optional boolean flag to display the tags_header element.
-                       Defaults to True. Adds class 'tags__hide-header' if falsy
-                       value passed (e.g., '', False, None).
+   hide_header:        The optional boolean flag to hide the tags_header element.
```

Comment:
Looks like the implementation also needs an update on https://github.com/cfpb/cfgov-refresh/pull/340/files#diff-3bfec8487d6ebccf797af7f1b22d6038R62 and https://github.com/cfpb/cfgov-refresh/pull/340/files#diff-1a0bc04460357f27c36992e161ec5be3R46